compiam/rhythm/transcription/__init__.py

```diff
@@ -382,3 +388,303 @@ def get_generators(
         )
 
         return training_generator, validation_generator
+
+
+class MnemonicTranscription:
```

Comment:
I understand both tools "transcribe" but I am wondering if it makes sense to put them in the same file. I think this may be a source of ambiguity, for usage and for new contributions. Should therefore melodia and FTANet go to the same file called `pitch_extraction`? Of course there is no right/wrong way to do it, but we should be consistent.

Reply:
good question, not sure. This does tabla transcription too, so since we have a module named tabla_transcription it makes sense to put it there. However it is not limited to tabla, it also does mridangam transcription. so I changed the name to transcription...

Reply:
I  dunno, I think honestly since there are so few projects at the moment it doesn't make much of a difference. However, going forward, structure will become more important and we should group similar tools sensibly.

Ultimately these are both transcription tools and very different from, for example, drum synthesis tools (which we don't have any of). In the ideal future where we have 100 tools in rhythm, it will make sense to group these together. And have separate pages dedicated to the separate tasks and why they are important in IAM analysis.